include/chopper/layout/configuration.hpp

```diff
@@ -39,6 +44,30 @@ struct configuration
     bool force_all_binnings{false};
     bool output_statistics{false};
     bool debug{false};
+
+private:
+    friend class cereal::access;
+
```

Comment:
We might be happy about it later if we add a version now:

For example: https://github.com/seqan/raptor/blob/master/include/raptor/index.hpp
The index has a `static constexpr uint32_t version{1u};`; the cereal functions then check validity.

Because we do not have a release right now, it's not so important; but it might pay off later.

Reply:
Good point!